featurebyte/service/preview.py

```diff
@@ -256,21 +256,32 @@ async def describe(
             columns_batch_size=columns_batch_size,
             total_num_rows=total_num_rows,
         )
-        df_queries = []
-        for describe_query in describe_queries.queries:
-            logger.debug("Execute describe SQL", extra={"describe_sql": describe_query.sql})
-            result = await session.execute_query_long_running(describe_query.sql)
-            columns = describe_query.columns
-            assert result is not None
-            df_query = pd.DataFrame(
-                result.values.reshape(len(columns), -1).T,
-                index=describe_query.row_names,
-                columns=[str(column.name) for column in columns],
+        await session.create_table_as(
+            table_details=describe_queries.data.output_table_name,
+            select_expr=describe_queries.data.expr,
+        )
```

Comment:
Seems like there is an issue when writing invalid dates into spark delta table:
```
CREATE TABLE `__TEMP_SAMPLED_DATA_66A25AD002AF101FA4F82384`
USING DELTA
TBLPROPERTIES (
  'delta.columnMapping.mode'='name',
  'delta.minReaderVersion'='2',
  'delta.minWriterVersion'='5'
) AS
SELECT
  `date_col` AS `date_col`
FROM `spark_catalog`.`featurebyte_20240725220133_999912`.`TABLE_INVALID_DATES`
```
Should we skip `tests.integration.api.test_view_describe.test_describe_invalid_dates` for spark?

Reply:
If we pass `kind="VIEW"`, it works for spark.

Reply:
Create a branch to test: https://github.com/featurebyte/featurebyte/pull/2479

Reply:
Setting `spark.sql.parquet.int96RebaseModeInWrite=CORRECTED` according to the error message also seems to fix it.

Reply:
I think we want to create a table but not a view to ensure the sampling is done once.